src/App.js

```diff
@@ -1,25 +1,42 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+// import Footer3 from "components/footers/Footer3.js";
+import Home from "./pages/Home";
+import CatIndex from "./pages/CatIndex";
+import CatShow from "./pages/CatShow";
+import CatNew from "./pages/CatNew";
+import CatEdit from "./pages/CatEdit";
+import NotFound from "./pages/NotFound";
+import mockCats from "./mockCats.js";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cats: mockCats,
+    };
+  }
 
+  render() {
+    return (
+      <Router>
+        <Header />
+        <h1>
+          CAT TINDER! *change {" "}
+        </h1>
```

Comment:
Haha - this looks temporary but in this project App.js is only handling big-picture logic. Display content like this will likely live in the Header component.